frontend/src/components/Wheel/index.js

```diff
@@ -1,12 +1,39 @@
 import { useEffect, useState } from "react";
-import SpinButton from "./button";
 import styles from "./index.module.css";
+import "./index.css";
 
 const Wheel = () => {
   const [wheelInfo, setWheelInfo] = useState({});
+  const [rotate, setRotate] = useState("circle");
```

Comment:
I don't think this is a good idea, it's error prone, you need to know change the whole class name when toggling states and it hiddes what is doing (indirection).

I would suggest using a bool as state, `isRotating` and then using the package `classnames` to toggle the correct classes.